### ISSUE TITLE
[BUG] Forms controls must have labels & form elements

### ISSUE THREAD

**Issue by acovher2** [rokwire/illinois-app#5153]
**Describe the bug**
Each input, select, textarea, progress, meter and output element must have an accessible name using label elements.

The create a Group form and Create an Event forms are missing these.
**Please check all forms for these elements and ensure there are labels for screen readers.** 

 This renders app inaccessible with a showstopper to those that utilize screen readers. 

**To Reproduce**
1. While logged into the web app in Chrome, go to Create a group form
2. Use Bookmarklet tools to identify form areas
3. There are none. 
4. Additionally, using NVDA, you can TAB through and hear "edit blank" many times and other confusing instructions. 

**Expected behavior**
Everything visually present is auditorily present for Screen reader users. Forms should be usable by all audiences. 

**Actual behavior** 
Only a sighted user can use this form. Completely inaccessible to screen reader users. 

**Screenshots**

https://github.com/user-attachments/assets/b4d008ed-5ef1-4632-93f2-ee12726e429d

**Desktop (please complete the following information):**
PC:
•	OS: Windows 11
•	Device: Dell laptop
•	Browser: Chrome 135.0.7049.116/Firefox 138.0.1

**Additional context**

**Comment by acovher2** [comment]
![Image](https://github.com/user-attachments/assets/0cc23974-9a6b-420b-bf04-e124338cd1d5)

This form is missing labels too. This is another one where I will not be able to open a bug for everywhere in the app this is. A better approach would be for development to utilize a screen reader such as TalkBack or VoiceOver before releasing for testing.  This will help you catch a fair amount of labeling and focus order issues ahead of time before they are bugs.
